string_lab.js

```diff
@@ -0,0 +1,65 @@
+//#1 console.log(); the first character of a string
```

Comment:
Please revisit this assignment as it is crucial to be able to manipulate strings and using its properties and methods effectively. Read questions more carefully next time, make mental map of how you would approach the problem, and think of which methods to use. Keep practicing and keep coding.